### PR TITLE
[WIP] Add int8 docs

### PR DESCRIPTION
**Pull request by xinyu-intel**

### CODE REVIEW

**Comment by kuonangzhe** [comment]
Are there any methods/docs for quantizing/calibrating random gluoncv models into int8?

**Comment by xinyu-intel** [comment]
We will support more quantized models and calibration tools later. Thanks:)

**Comment by kuonangzhe** [comment]
@xinyu-intel That's great, look forward to the new progress!

File: docs/tutorials/deployment/int8_inference.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
+"""3. Inference with Quantized Models
+================================
```
Comment (zhreshold) [inline docs/tutorials/deployment/int8_inference.py]: make sure ==== is same length as title

File: docs/tutorials/deployment/int8_inference.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                                    eval mini-batch size
+            --dataset DATASET     eval dataset.
+            --num-workers NUM_WORKERS, -j NUM_WORKERS
+                                    Number of data workers
+            --num-gpus NUM_GPUS   number of gpus to use.
+            --pretrained PRETRAINED
+                                    Load weights from previously saved parameters.
+            --save-prefix SAVE_PREFIX
+                                    Saving parameter prefix
+"""
```
Comment (zhreshold) [inline docs/tutorials/deployment/int8_inference.py]: add a new line, I suspect this cause CI fail to parse the doc

**Comment by mli** [comment]
Job PR-691-3 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-691/3/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-691-4 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-691/4/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-691-5 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-691/5/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-691-7 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-691/7/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-691-8 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-691/8/index.html 
Code coverage of this PR:  vs. Master:

**Comment by xizi** [comment]
Does mxnet support float16 or int8 in CPU mode? When i train a floa16 model and test in CPU, it reports error : RuntimeError: [21:56:05] src/operator/contrib/../linalg_impl.h:165: FP16 gemm on cpu not implemented!

**Comment by zhreshold** [comment]
@xizi CPU currently don't support FP16, only int8 and fp32

**Pull request merged**